src/Form/EventGroupCategoryType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\EventCategory;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class EventGroupCategoryType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'class' => EventCategory::class,
+            'choice_label' => 'name',
+            'query_builder' => function (EntityRepository $er) {
+                return $er->createQueryBuilder('ec')
+                    ->leftJoin('ec.eventGroupCategory', 'egc')
+                    ->where('ec.status = :status')
+                    ->andWhere('egc.status = :status')
+                    ->orderBy('ec.eventGroupCategory', 'ASC')
+                    ->addOrderBy('ec.name', 'ASC')->setParameters([
+                        'status' => EventCategory::ENABLED,
+                    ]);
+            },
+            'group_by' => function ($category) {
+                /** @var EventCategory $category */
+                if ($category->getEventGroupCategory()) {
```

Comment:
why this test if the category group cannot be null ?

Reply:
removed